hail/src/main/scala/is/hail/annotations/UnsafeRow.scala

```diff
@@ -63,6 +63,7 @@ object UnsafeRow {
 
   def read(t: PType, region: Region, offset: Long): Any = {
     t match {
+      case PVoid => region.loadInt(offset)
```

Comment:
Let's keep `UnsafeRow.read` as-is and keep the void-handling in cxx.Compile/Emit, since the fact that we have to return something is an artifact of how that's structured.